content/root/platform-customization.textile

```diff
@@ -120,17 +120,22 @@ Repeat this step for all fallback hosts in the @fallbackHosts@ array, if provide
 
 h3. Modify your code to use your custom environment
 
-Now that you have tested your custom endpoints, you must modify your code to instantiate the Ably client with the provided @ClientOptions@.
+Now that you have tested your custom endpoints, you must modify your code to instantiate the Ably client with the provided @ClientOptions@. For more information about the clientOptions object, see "clientOptions":/api/rest-sdk#client-options.
 
 The following demonstrates how you might do this using the JavaScript SDK and an Ably subdomain. Even though the sample code is shown in JavaScript, the same options are common to all the client library SDKs:
```

Comment:
This texts says it's showing what to set for an Ably subdomain, however the code is now showing options for a customer's own domain. I wonder if it's worth showing an example of each?

Reply:
@m-hulbert, please see: https://github.com/ably/docs/commit/97a9abcc2f30353a1bd67f63d71582295b7bc787